src/Microsoft.Azure.WebJobs.Host/Listeners/HostListenerFactory.cs

```diff
@@ -64,6 +58,12 @@ public async Task<IListener> CreateAsync(CancellationToken cancellationToken)
                     continue;
                 }
 
+                IListenerFactory listenerFactory = functionDefinition.ListenerFactory;
```

Comment:
Why this move?

Reply:
we want to log disabled function if there is no listener